doc/api.md

```diff
@@ -448,6 +448,19 @@ This is a flag that can be enabled by activating the parameter `appendMode` in t
 activated, the update requests to the Context Broker will be performed always with APPEND type, instead of the default
 UPDATE. This have implications in the use of attributes with Context Providers, so this flag should be used with care.
 
+When running the agent using `appendMode=false`, if sending measures that are not included in the config group (as 
+active measures), the IoT Agent should return a `422 Unprocessable Entity` code with the following message:
```

Comment:
```suggestion
active measures), the IoT Agent returns a `422 Unprocessable Entity` code with the following message:
```

As I think is a description of functional behaviour and optionality is not possible here (I could be wrong).